### PR TITLE
[_]: Fix/update subscription to lifetime

### PR DESCRIPTION
**Pull request by xabg2**
Now, the user can buy a lifetime plan even if he has an active subscription.

30 seconds waiting time added to the banner.

### CODE REVIEW

File: src/app/banners/BannerWrapper.tsx [hunk 1]
```diff
@@ -2,15 +2,19 @@ import { useEffect, useState } from 'react';
...
 
   useEffect(() => {
-    if (localStorage.getItem('showLifetimeBanner') === 'false') {
+    if (!localStorage.getItem('showLifetimeBanner')) {
+      setTimeout(() => {
+        setShowBanner(true);
+      }, 30000);
+    } else if (localStorage.getItem('showLifetimeBanner') === 'false') {
       setShowBanner(false);
     }
```
Comment (CandelR) [inline src/app/banners/BannerWrapper.tsx]: Now that it is closed by default, is it necessary to check if the value of the local storage is false when the component is mounted? I think this logic is no longer necessary

File: src/app/core/views/Preferences/tabs/Plans/PlanSelector.tsx [hunk 2]
```diff
@@ -66,19 +66,52 @@ export default function PlanSelector({ className = '' }: { className?: string })
...
+              notificationsService.show({
+                type: ToastType.Success,
+                text: 'Payment successful',
+              });
+            }
+          });
+        } catch (error) {
+          console.error(error);
+          notificationsService.show({
+            text: 'Something went wrong while updating your subscription',
```
Comment (CandelR) [inline src/app/core/views/Preferences/tabs/Plans/PlanSelector.tsx]: Add this to translations files in order to manage all languages

File: src/app/payment/views/CheckoutView/CheckoutPlanView.tsx [hunk 3]
```diff
@@ -74,16 +74,49 @@ export default function CheckoutPlanView(): JSX.Element {
...
+              notificationsService.show({
+                type: ToastType.Success,
+                text: 'Payment successful',
+              });
+            }
+          });
+        } catch (error) {
+          console.error(error);
+          notificationsService.show({
+            text: 'Something went wrong while updating your subscription',
```
Comment (CandelR) [inline src/app/payment/views/CheckoutView/CheckoutPlanView.tsx]: Same here

**Review by CandelR — APPROVED** [review]

File: src/app/core/views/Preferences/tabs/Plans/PlanSelector.tsx (hunk 2 above)
Comment (xabg2) [reply src/app/core/views/Preferences/tabs/Plans/PlanSelector.tsx]: On it

**Pull request merged**